corehq/apps/linked_domain/migrations/0016_auto_20210720_1908.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.2.24 on 2021-07-20 19:08
```

Comment:
Nitpick: human-readable names for migrations are always good.

...I was going to suggest looking at other files in the directory, but they're terrible. Maybe something like `0016_add_tableau_choice`?